File: app/src/main/java/com/brickgit/imagesearch/model/Count.java
package com.brickgit.imagesearch.model;

public class Count {

	private String mCount;

	public Count(int count) {
		if (count < 1000) {
			mCount = String.valueOf(count);
		}
		else {
			mCount = String.format("%dK", count / 1000);
		}
	}

	public String getCount() {
		return mCount;
	}
}

File: app/src/main/java/com/guanhaolin/pearch/model/Count.kt
package com.guanhaolin.pearch.model

class Count(count: Int) {
    var count: String? = null

    init {
        if (count < 1000) {
            this.count = count.toString()
        } else {
            this.count = String.format("%dK", count / 1000)
        }
    }
}
